pallets/xcmp-queue/src/lib.rs

```diff
@@ -146,6 +147,7 @@ pub mod pallet {
 	/// case of the need to send a high-priority signal message this block.
 	/// The bool is true if there is a signal message waiting to be sent.
 	#[pallet::storage]
+	#[pallet::storage_prefix = "OutboundXcmpStatus"]
```

Comment:
is the name here really that important to change it and have this code?

I really think we should separate this change from the rest, and then make a separate decision.